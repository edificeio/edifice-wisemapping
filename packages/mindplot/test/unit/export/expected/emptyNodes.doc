<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Word.Document"?><w:wordDocument xmlns:sl="http://schemas.microsoft.com/schemaLibrary/2003/core" xmlns:w="http://schemas.microsoft.com/office/word/2003/wordml" xmlns:w10="urn:schemas-microsoft-com:office:word" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882" xmlns:wx="http://schemas.microsoft.com/office/word/2003/auxHint" xmlns:aml="http://schemas.microsoft.com/aml/2001/core" xmlns:o="urn:schemas-microsoft-com:office:office">
<o:DocumentProperties>
<o:Title/>
</o:DocumentProperties>
<w:styles>
<w:versionOfBuiltInStylenames w:val="4"/>
<w:latentStyles w:defLockedState="off" w:latentStyleCount="156"/>
<w:style w:type="paragraph" w:default="on" w:styleId="Normal">
<w:name w:val="Normal"/>
<w:rsid w:val="00831C9D"/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:sz w:val="24"/>
<w:sz-cs w:val="24"/>
<w:lang w:val="EN-US" w:fareast="EN-US" w:bidi="AR-SA"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading1">
<w:name w:val="heading 1"/>
<wx:uiName wx:val="Heading 1"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading1"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="32"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading2">
<w:name w:val="heading 2"/>
<wx:uiName wx:val="Heading 2"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading2"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="1"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading3">
<w:name w:val="heading 3"/>
<wx:uiName wx:val="Heading 3"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading3"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="2"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading4">
<w:name w:val="heading 4"/>
<wx:uiName wx:val="Heading 4"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading4"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="3"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading5">
<w:name w:val="heading 5"/>
<wx:uiName wx:val="Heading 5"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading5"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="4"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading6">
<w:name w:val="heading 6"/>
<wx:uiName wx:val="Heading 6"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading6"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="5"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading7">
<w:name w:val="heading 7"/>
<wx:uiName wx:val="Heading 7"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading7"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="6"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading8">
<w:name w:val="heading 8"/>
<wx:uiName wx:val="Heading 8"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading8"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="7"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:i/>
<w:i-cs/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading9">
<w:name w:val="heading 9"/>
<wx:uiName wx:val="Heading 9"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading9"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="8"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="character" w:default="on" w:styleId="DefaultParagraphFont">
<w:name w:val="Default Paragraph Font"/>
<w:semiHidden/>
</w:style>
<w:style w:type="table" w:default="on" w:styleId="TableNormal">
<w:name w:val="Normal Table"/>
<wx:uiName wx:val="Table Normal"/>
<w:semiHidden/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
<w:tblPr>
<w:tblInd w:w="0" w:type="dxa"/>
<w:tblCellMar>
<w:top w:w="0" w:type="dxa"/>
<w:left w:w="108" w:type="dxa"/>
<w:bottom w:w="0" w:type="dxa"/>
<w:right w:w="108" w:type="dxa"/>
</w:tblCellMar>
</w:tblPr>
</w:style>
<w:style w:type="list" w:default="on" w:styleId="NoList">
<w:name w:val="No List"/>
<w:semiHidden/>
</w:style>
<w:style w:type="paragraph" w:styleId="Title">
<w:name w:val="Title"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Title"/>
<w:spacing w:before="240" w:after="60"/>
<w:jc w:val="center"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="28"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="BodyText">
<w:name w:val="Body Text"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="BodyText"/>
<w:spacing w:after="120"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
</w:styles>
<w:body>
<wx:sect>
    <wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Title"/>
</w:pPr>
<w:r>
<w:t/>
</w:r>
</w:p>
<w:p>
<w:pPr>
<w:pStyle w:val="BodyText"/>
</w:pPr>
<w:r>
<w:t>
                    
                    
                </w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>objectifs journée</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>"business plan" associatif ?</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>modèle / activités responsabilités</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>articulations / LOG</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>SWOT</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t/>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>l'entreprise a aujourd'hui un potentiel important</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>compétences professionnel</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>citoyen</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>forte chance de réussite</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>apporter des idées et propsitions à des questions sociétales</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>notre manière d"y répondre avec notamment les technlogies</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>l'opportunité et la demande sont fortes aujourd'hui, avec peu de "concurrence"</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>ensemble de ressources "rares"</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>capacités de recherche et innovation</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>motivation du groupe et sens partagé entre membres</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>professionnellement : expérience collective et partage d'outils en pratique</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>ouverture vers mode de vie attractif perso / pro</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>potentiel humain, humaniste et citoyen</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>assemblage entre atelier et outillage</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>capacité de réponder en local et en global</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>associatif : contxte de crise multimorphologique / positionne référence en réflexion et usages</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>réseau régional et mondial de l'économie de la ,connaisance</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>asso prend pied dans le monde de la recherche</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>labo de l'innovation sociopolitique</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>acteur valable avec pouvoirs et acteurs en place</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>autonomie par prestations et services</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>triptique</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>éthique de la discussion</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>pari de la délégation</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>art de la décision</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>réussir à caler leprojet en adéquation avec le contexte actuel</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>assoc : grouper des personnes qui développent le concept</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>traduire les belles pensées au niveau du citoyen</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>compréhension</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>adhésion</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>ressources contributeurs réfréents</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>reconnaissance et référence exemplaires</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>financeements suffisants pour bien exister</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>notre organisation est claire</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>prendre des "marchés émergent"</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>double stratup avec succes-story</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>engageons une activité présentielle forte, conviviale et exemplaire</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>attirer de nouveaux membres locomotives</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>pratiquons en interne et externe une gouvernance explaire etune citoyennté de rêve</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Risques : cauchemars, dangers</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>disparition des forces vives, départ de membres actuels</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>opportunités atteignables mais difficile</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>difficultés de travailler ensemble dans la durée</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>risque de rater le train</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>sauter dans le dernier wagon et rester à la traîne</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>manquer de professionnalisme</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>perte de crédibilité</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>s'isoler entre nous et perdre le contact avec les autres acteurs</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>perdre la capacité de réponse au global</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>manque de concret, surdimension des reflexions</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>manque d'utilité socioplolitique</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>manque de nouveaux membres actifs, fidéliser</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>faire du surplace et</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>manque innovation</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t/>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>ne pas vivre ce que nous affirmons</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>cohérence entre langage gouvernance et la pratique</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>groupe de base insuffisant</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>non attractifs / nouveaux</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading4"/>
</w:pPr>
<w:r>
<w:t>pas ennuyants</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>pas efficaces en com</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>trop lent, rater l'opportunité actuelle</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>débordés par "concurrences"</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>départs de didier, micvhel, rené, corinne MCD etc</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>conclits de personnes et schisme entre 2 groupes ennemis</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>groupe amicale mais très merdique</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>système autocratique despotique ou sectaire</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t/>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
</wx:sub-section>
</wx:sub-section>
</wx:sect>
</w:body>
</w:wordDocument>
