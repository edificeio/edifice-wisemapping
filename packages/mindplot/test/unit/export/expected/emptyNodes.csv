 
,
            objectifs journée 
,
            ,
            "business plan" associatif ? 
,
            ,
            modèle / activités responsabilités 
,
            ,
            articulations / LOG 
,
            SWOT 
,
            ,
             
,
            ,
            ,
            l'entreprise a aujourd'hui un potentiel important 
,
            ,
            ,
            ,
            compétences professionnel 
,
            ,
            ,
            ,
            citoyen 
,
            ,
            ,
            ,
            forte chance de réussite 
,
            ,
            ,
            apporter des idées et propsitions à des questions sociétales 
,
            ,
            ,
            notre manière d"y répondre avec notamment les technlogies 
,
            ,
            ,
            l'opportunité et la demande sont fortes aujourd'hui, avec peu de "concurrence" 
,
            ,
            ,
            ensemble de ressources "rares" 
,
            ,
            ,
            capacités de recherche et innovation 
,
            ,
            ,
            motivation du groupe et sens partagé entre membres 
,
            ,
            ,
            professionnellement : expérience collective et partage d'outils en pratique 
,
            ,
            ,
            ouverture vers mode de vie attractif perso / pro 
,
            ,
            ,
            potentiel humain, humaniste et citoyen 
,
            ,
            ,
            assemblage entre atelier et outillage 
,
            ,
            ,
            capacité de réponder en local et en global 
,
            ,
            ,
            associatif : contxte de crise multimorphologique / positionne référence en réflexion et usages 
,
            ,
            ,
            réseau régional et mondial de l'économie de la ,connaisance 
,
            ,
            ,
            asso prend pied dans le monde de la recherche 
,
            ,
            ,
            labo de l'innovation sociopolitique 
,
            ,
            ,
            acteur valable avec pouvoirs et acteurs en place 
,
            ,
            ,
            autonomie par prestations et services 
,
            ,
            ,
            triptique 
,
            ,
            ,
            ,
            éthique de la discussion 
,
            ,
            ,
            ,
            pari de la délégation 
,
            ,
            ,
            ,
            art de la décision 
,
            ,
            ,
            réussir à caler leprojet en adéquation avec le contexte actuel 
,
            ,
            ,
            assoc : grouper des personnes qui développent le concept 
,
            ,
            ,
            traduire les belles pensées au niveau du citoyen 
,
            ,
            ,
            ,
            compréhension 
,
            ,
            ,
            ,
            adhésion 
,
            ,
            ,
            ressources contributeurs réfréents 
,
            ,
            ,
            reconnaissance et référence exemplaires 
,
            ,
            ,
            financeements suffisants pour bien exister 
,
            ,
            ,
            notre organisation est claire 
,
            ,
            ,
            prendre des "marchés émergent" 
,
            ,
            ,
            double stratup avec succes-story 
,
            ,
            ,
            engageons une activité présentielle forte, conviviale et exemplaire 
,
            ,
            ,
            attirer de nouveaux membres locomotives 
,
            ,
            ,
            pratiquons en interne et externe une gouvernance explaire etune citoyennté de rêve 
,
            ,
            Risques : cauchemars, dangers 
,
            ,
            ,
            disparition des forces vives, départ de membres actuels 
,
            ,
            ,
            opportunités atteignables mais difficile 
,
            ,
            ,
            difficultés de travailler ensemble dans la durée 
,
            ,
            ,
            risque de rater le train 
,
            ,
            ,
            sauter dans le dernier wagon et rester à la traîne 
,
            ,
            ,
            manquer de professionnalisme 
,
            ,
            ,
            ,
            perte de crédibilité 
,
            ,
            ,
            s'isoler entre nous et perdre le contact avec les autres acteurs 
,
            ,
            ,
            perdre la capacité de réponse au global 
,
            ,
            ,
            manque de concret, surdimension des reflexions 
,
            ,
            ,
            manque d'utilité socioplolitique 
,
            ,
            ,
            manque de nouveaux membres actifs, fidéliser 
,
            ,
            ,
            faire du surplace et 
,
            ,
            ,
            ,
            manque innovation 
,
            ,
            ,
            ,
             
,
            ,
            ,
            ne pas vivre ce que nous affirmons 
,
            ,
            ,
            ,
            cohérence entre  langage gouvernance et la pratique 
,
            ,
            ,
            groupe de base insuffisant 
,
            ,
            ,
            non attractifs / nouveaux 
,
            ,
            ,
            ,
            pas ennuyants 
,
            ,
            ,
            pas efficaces en com 
,
            ,
            ,
            trop lent, rater l'opportunité actuelle 
,
            ,
            ,
            débordés par "concurrences" 
,
            ,
            ,
            départs de didier, micvhel, rené, corinne MCD etc 
,
            ,
            ,
            conclits de personnes et schisme entre 2 groupes ennemis 
,
            ,
            ,
            groupe amicale mais très merdique 
,
            ,
            ,
            système autocratique despotique ou sectaire 
,
            ,
            ,
             
